JS1/assignment1/Home.html

```diff
@@ -0,0 +1,87 @@
+            </p>
+            <p>
+              Using your word processor, you can create headings, write in paragraphs, insert pictures, or underline text. Similarly, you can use HTML to structure and style text that appears on websites.Markup language documents, like HTML documents, are just plain text files. Unlike documents created with a word processor, you can view an HTML file using any web browser on any type of computer.
+            </p>
+          </td>
+        </tr>
+    </table>
+
+    <br><br><br><br>
+
+    <table id="how_it_works" align="center" height="40%" width="90%">
+      <tr>
+        <td align="center" colspan ="3" height="5%"><h2><b>How it works?</b></h2></td>
+      </tr>
+      <tr>
+        <td align="center" height="30%"><img src="HTML logo.png"/></td>
+        <td align="center" height="30%"><img src="HTML logo.png"/></td>
+        <td align="center" height="30%"><img src="HTML logo.png"/></td>
+      </tr>
+      <tr><td colspan="3"></td></tr>
+      <tr>
+        <td align="center" height="5%"><h3>Name1</h3></td>
+        <td align="center" height="5%"><h3>Name2</h3></td>
+        <td align="center" height="5%"><h3>Name3</h3></td>
+      </tr>
+
+      <tr>
+        <td align="center" height="30%">Name and some long description of who the founding fathers of the company are. What their contributions are. Espically, their position and where they are.</td>
+        <td align="center" height="30%">Name and some long description of who the founding fathers of the company are. What their contributions are. Espically, their position and where they are.</td>
+        <td align="center" height="30%">Name and some long description of who the founding fathers of the company are. What their contributions are. Espically, their position and where they are.</td>
+      </tr>
+    </table>
+
+    <br><hr>
+
+    <table id="footer" align ="center" height="2%" width="90%">
+      <tr>
+        <td align="center"><p style="color: grey">@ Copyright 2016-17 | All Rights Reserved | Metacube Training</p></td>
```

Comment:
Use li tag to create "@ Copyright 2016-17 | All Rights Reserved | Metacube Training" .